route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -6624,19 +6626,19 @@ public TerritoryZone UpdateTerritory(TerritoryZoneParameters territoryParameters
                             if (response.IsSuccessStatusCode && response.StatusCode == HttpStatusCode.OK ||
                                 response.StatusCode == HttpStatusCode.RedirectMethod)
                             {
-                                var streamTask = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                                    var streamTask = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
```

Comment:
is there any change or it is unintended formatting change?

Reply:
I first changed it, but then I saw that it did not make sense, so I restored the code, and unintended formatting happened.